kie-api/src/main/java/org/kie/api/definition/process/Node.java

```diff
@@ -31,6 +31,17 @@
      */
     long getId();
 
+    /**
+     * The node id provided in the bpmn 
+     */
+    String getUniqueId();
```

Comment:
This is probably going to confuse users, as it's two similar fields with similar description even.  getUniqueId() is an existing operation in the internal Node interface already, that simply concatenates the nodeId with the nodeIds of the parent.  This is somewhat outdated, as we later switched how node ids (long) are generated so they are unique across a process, so I think this operation is outdated.
I would recommend to keep getUniqueId() unchanged (for backwards compatibility) but keep it in internal node interface only, not promote it to kie-api.  That way we would only have getNodeUniqueId() here, wdyt?

Reply:
fine.